src/routes/transactions/mappers/multisig-transactions/multisig-transaction-execution-details.mapper.ts

```diff
@@ -85,45 +84,44 @@ export class MultisigTransactionExecutionDetailsMapper {
   }
 
   /**
-   * Gets an array of {@link AddressInfo} representing the confirmations of the replacement
+   * Gets an array of {@link AddressInfo} representing the confirmations of the rejection
    * transaction for the {@link Transaction} passed in.
    *
-   * When a transaction is cancelled, another transaction replaces it, having the same nonce
-   * and isExecuted attribute set to true. This function retrieves that transaction, and
-   * collects its confirmations as rejectors.
+   * When a transaction is cancelled, another transaction acts as rejection.
+   * The rejection transaction has the same nonce of the transaction being cancelled,
+   * its value is 0, and it destination is the address of the safe that owns the transaction.
+   *
+   * This function retrieves that rejection transaction, and collects its confirmations as rejectors.
    *
    * @param chainId - chain id to use
    * @param transaction - transaction to use
    * @param safe - safe to use
-   * @returns confirmations for the replacement transaction
+   * @returns confirmations for the rejection transaction
    */
   private async _getRejectors(
```

Comment:
Here is the main fix. Besides changing the inner logic, I've also done two small amendments: 
- I've changed the naming from `replacement*` to `rejection*`, I think it's clearer.
- I've removed the `loggingService.debug` call: I think this will be called every time the function is executed and has no value. In fact, in the old implementation, [the service doesn't log anything](https://github.com/safe-global/safe-client-gateway/blob/eb481938a49c29b979fe98a5577f719f42a2b2eb/src/utils/transactions.rs#L130) for a not-found rejection transaction (which is the most common case).